lib/cc/cli/init.rb

```diff
@@ -10,7 +10,7 @@ class Init < Command
       def run
         if !upgrade? && filesystem.exist?(CODECLIMATE_YAML)
           warn "Config file .codeclimate.yml already present.\nTry running 'validate-config' to check configuration."
-          create_default_configs
+          create_default_engine_configs
```

Comment:
The commits don't seem sliced that way. https://github.com/codeclimate/codeclimate/commit/668956f7ce32d996f27836d7d1aaf82649f876fb has a rename in it, for example. An, in that commit, I still can't see where you fallback to config-generator when `existing_cc_config` *doesn't* exist, but maybe I'm misunderstanding the intent (because of the diff confusion?).